Sources/Classes/Views/CustomizableButton/CustomizableButtonView.swift

```diff
@@ -0,0 +1,355 @@
+        }
+        didSet {
+            if spinnerView != nil {
+                addSpinner()
+            }
+        }
+    }
+
+    public var appearance = Appearance() {
+        didSet {
+            configureAppearance()
+            configureConstraints()
+        }
+    }
+
+    public var buttonIsDisabledWhileLoading = false
+    public var hidesLabelWhenLoading = false
+
+    // MARK: - Computed Properties
+
+    public var tapObservable: Observable<Void> {
+        return button.rx.tap.asObservable()
+    }
+
+    override open var forFirstBaselineLayout: UIView {
+        return button.forFirstBaselineLayout
+    }
+
+    override open var forLastBaselineLayout: UIView {
+        return button.forLastBaselineLayout
+    }
+
+    override open func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        if var touchPoint = touches.first?.location(in: self) {
+            touchPoint = convert(touchPoint, to: self)
+            if button.frame.contains(touchPoint) && !button.isEnabled {
+                tapOnDisabledButton?()
+            }
+        }
+        super.touchesBegan(touches, with: event)
+    }
+
+    // MARK: - Initialization
+
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        initializeView()
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initializeView()
+    }
+
+    // MARK: - UI
+
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+        if shadowView.layer.cornerRadius == 0 {
+            shadowView.layer.shadowPath = UIBezierPath(rect: button.bounds).cgPath
+        }
+    }
+
+    private func set(active: Bool) {
+        button.isEnabled = buttonIsDisabledWhileLoading ? !active : true
+        
+        if hidesLabelWhenLoading {
+            button.titleLabel?.layer.opacity = active ? 0 : 1
+        }
+
+        if active {
+            spinnerView?.isHidden = false
+            spinnerView?.startAnimating()
+        } else {
+            spinnerView?.isHidden = true
+            spinnerView?.stopAnimating()
+        }
+    }
+
+    private func addSpinner() {
+        if let spinner = spinnerView {
+            addSubview(spinner)
+            configureSpinnerConstraints()
+            spinner.isHidden = true
+        }
+    }
+
+    private func removeSpinner() {
+        if spinnerView != nil {
+            self.spinnerView?.removeFromSuperview()
+            self.spinnerView = nil
+        }
+    }
+
+    // MARK: - Layout
+
+    private func configureConstraints() {
+        button.pinToSuperview(with: appearance.buttonInsets)
+        configureShadowViewConstraints()
+    }
+
+    private func configureSpinnerConstraints() {
+        spinnerView?.translatesAutoresizingMaskIntoConstraints = false
+        switch appearance.spinnerPosition {
+        case .center:
+            spinnerView?.centerXAnchor.constraint(equalTo: button.centerXAnchor).isActive = true
```

Comment:
try to use NSLayoutConstraints.activate()

Reply:
fixed